src/main/java/n1/variables/operators/statements/expressions/Task1.java

```diff
@@ -0,0 +1,56 @@
+package n1.variables.operators.statements.expressions;
+
+import java.util.Scanner;
+
+public class Task1 {
+
+    public static void main(String[] args) {
+        System.out.println("Enter a number and click Enter:");
+        Scanner scaner = new Scanner(System.in);
+        String switchCondition = "";
+
+        // first solution:
+        int number = scaner.nextInt();
+
+        if (number < 100){
+            switchCondition = "too less";
+            System.out.println("too less");
+        }
+
+        if (number >= 100 && number <= 100500){
+            switchCondition = "its okay";
+            System.out.println("its okay");
+        }
+
+        if (number > 100500){
+            switchCondition = "too big";
+           System.out.println("too big");
+        }
+        System.out.println("==================================");
+
+        // second solution:
+        if (number < 100){
+            System.out.println("too less");
+        }else if (number >= 100 && number <= 100500){
+            System.out.println("its okay");
+        }else{
+            System.out.println("too big");
+        }
+        System.out.println("==================================");
+
+        // third solution:
+        switch (switchCondition) {
```

Comment:
use input number for a switch, don't forget about default statement

Reply:
can't check a range of values via switch statement

Reply:
switch statement has 'default'. Please, read about it and refactor code to use switch.